ui/core/src/model/trace-data.ts

```diff
@@ -0,0 +1,29 @@
+// source: https://grafana.com/docs/tempo/latest/api_docs/#search
+
+export interface TraceData {
```

Comment:
The data-model exposed here is not necessary correlated with a particular datasource. It's more the data model that will be used by the panel that will display the traces.

For example, if we are going to support Jaeger, the data exposed (I supposed) won't be the same than Tempo but we will want to reuse the TraceChart.

If you look at what has been done for the `TimeSeries`, you have a `TimeSeries` data model but then if you look at the package `prometheus-plugin`, you have the actual data model of the Prometheus API and you will notice that it is slightly different than the `timeseries` data model.

Reply:
Ah, that makes sense. 
In the latest push I updated `TraceData` to be more generic (in /ui/core/src/model/trace-data.ts) and created a separate model for a Tempo datasource (in /ui/core/src/model/api-type.ts).